app/assets/stylesheets/hackney_colour.scss

```diff
@@ -4,11 +4,11 @@ $hackney-colours: (
   "housing-3": #2dccd3,
   "status-planned": #009ca6,
   "status-complete": darkgray,
-  "status-not-complete": crimson
+  "status-not-complete": crimson,
+  "hackney-warning": #FFA400
```

Comment:
Prefixing with `hackney` here is redundant. `hackney-colour("hackney-warning")` has too much hackney in it.